tests/cases/standalone/common/flow/flow_basic.sql

```diff
@@ -199,8 +199,6 @@ where
 
 SHOW CREATE FLOW filter_numbers_basic;
 
-SHOW CREATE TABLE out_num_cnt_basic;
```

Comment:
We can remove all these tests (`test interprete interval`):
https://github.com/GreptimeTeam/greptimedb/blob/9af41600687ebaee837694d50bf1668bae5ffb1f/tests/cases/standalone/common/flow/flow_basic.sql#L175-L208